r/R/dplyr-summarize.R

```diff
@@ -161,6 +166,18 @@ summarize_eval <- function(name, quosure, ctx, recurse = FALSE) {
     return()
   }
 
+  if ("median" %in% funs_in_expr) {
```

Comment:
Make a comment up here explaining that for median/quantile, we have to translate the function into something that does a `mutate()` after aggregating, and we need to do that before determining outer_agg/inner_agg below.

Reply:
Done in c11ab71